app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsTasksFragment.kt

```diff
@@ -136,7 +137,10 @@ class SuggestedEditsTasksFragment : Fragment() {
         binding.swipeRefreshLayout.setOnRefreshListener { refreshContents() }
 
         binding.errorView.retryClickListener = View.OnClickListener { refreshContents() }
-        binding.errorView.loginClickListener = View.OnClickListener { requestLogin.launch(LoginActivity.newIntent(requireContext(), LoginActivity.SOURCE_SUGGESTED_EDITS)) }
+        binding.errorView.loginClickListener = View.OnClickListener {
+            ContributionsDashboardEvent.logAction("login_click", "contrib_dashboard${ContributionsDashboardEvent.anonSuffix()}")
```

Comment:
Doesn't the schema already have an automatic field for `is_anon` and `is_temp`? (Why do we need extra logic to append `_anon` here, too?)

Reply:
Done!